build.gradle

```diff
@@ -86,6 +87,14 @@ subprojects {
         }
     }
 
+    idea{
```

Comment:
IntelliJ doesn't play nice with Gradle when it comes to custom sourceSets. The best I could come up with is using the idea plugin. If this isn't worth adding another plugin, I can remove it.